packages/core/src/sfdxwrappers/CreateSourcePackageImpl.ts

```diff
@@ -114,6 +123,78 @@ export default class CreateSourcePackageImpl {
     return this.packageArtifactMetadata;
   }
 
+  private handleApexTestClasses(mdapiPackage: any) {
+    let apexTypeFetcher: ApexTypeFetcher = new ApexTypeFetcher();
+    let classTypes;
+    try {
+      classTypes = apexTypeFetcher.getApexTypeOfClsFiles(path.join(mdapiPackage.mdapiDir, `classes`));
+    }
+    catch (erorr) {
```

Comment:
typo

Reply:
Whats the typo?

Reply:
Ah got it

Reply:
Should we also log the error message here. For example if the directory doesn't exist?

Reply:
No as source packages without apex are completely okie